src/UnloadCopyUtility/redshift_unload_copy.py

```diff
@@ -75,16 +75,36 @@ def __init__(self,
         # load the configuration
         self.config_helper = ConfigHelper(config_file, self.s3_helper)
 
-        source = ResourceFactory.get_source_resource_from_config_helper(self.config_helper, self.region)
-
-        destination = ResourceFactory.get_target_resource_from_config_helper(self.config_helper, self.region)
-
         self.task_manager = TaskManager()
         self.barrier_after_all_cluster_pre_tests = NoOperationTask()
         self.task_manager.add_task(self.barrier_after_all_cluster_pre_tests)
         self.barrier_after_all_resource_pre_tests = NoOperationTask()
         self.task_manager.add_task(self.barrier_after_all_resource_pre_tests)
 
+        src_config = self.config_helper.config['unloadSource']
+        dest_config = self.config_helper.config['copyTarget']
+        if(src_config['tableNames']):
```

Comment:
Shouldn't this have been `if ('tableNames' in src_config)`? Single table migration does not appear to be working